pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[tool.ruff]
+select = [
+  "A",    # flake8-builtins
+  "ARG",  # flake8-unused-arguments
+  "B",    # flake8-bugbear
+  "C4",   # flake8-comprehensions
+  "C90",  # McCabe cyclomatic complexity
+  "E",    # pycodestyle
+  "EM",   # flake8-errmsg
+  "EXE",  # flake8-executable
+  "F",    # Pyflakes
+  "G",    # flake8-logging-format
+  "I",    # isort
+  "ICN",  # flake8-import-conventions
+  "INT",  # flake8-gettext
+  "ISC",  # flake8-implicit-str-concat
+  "PGH",  # pygrep-hooks
+  "PIE",  # flake8-pie
+  "PL",   # Pylint
+  "PT",   # flake8-pytest-style
+  "PYI",  # flake8-pyi
+  "RET",  # flake8-return
+  "RSE",  # flake8-raise
+  "RUF",  # Ruff-specific rules
+  "S",    # flake8-bandit
+  "SIM",  # flake8-simplify
+  "SLF",  # flake8-self
+  "T10",  # flake8-debugger
+  "TCH",  # flake8-type-checking
+  "TID",  # flake8-tidy-imports
+  "W",    # pycodestyle
+  "YTT",  # flake8-2020
+]
+ignore = [
+  "B904",
```

Comment:
Are this ignores TODOs or forever things? Sometimes useful to document why/how the might be fixed if someone has an interest to try and address them later

Reply:
I will let others determine if they want to reduce the ignore list

Rule information is easy to acquire: `ruff rule B904`
# raise-without-from-inside-except (B904)

Derived from the **flake8-bugbear** linter.

Message formats:
* Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling